docs/OV_Runtime_UG/precision_control.md

```diff
@@ -0,0 +1,31 @@
+# Precision Control {#openvino_docs_OV_UG_Precision_Control}
+
+@sphinxdirective
+
+Regardless of IR precision, devices will run in high performance mode by default. For GPU this means ``fp16`` inference and for CPU - ``bf16`` inference (if available). Previously, you had to convert the IR to ``fp16`` for the GPU to run in ``fp16``, and for the CPU, it was high precision by default. Now the devices have been aligned and this selection has been disconnected from IR precision. If high performance is causing the accuracy issue (only seen a few times in history), you can use the ``inference_precision`` hint and set it to accuracy.
+
+Separately, you can control the IR precision. Essentially the IR precision becomes a way to compress your model by reducing the precision of the weights, and it doesn't affect how devices execute the model.
+
+This change clears up a lot of confusion where, for example, you couldn't execute a high-performance model on the GPU by default, and the behavior between devicess was different.
+
```

Comment:
I'd add few words on why usage of lower precision is important for the performance: compute bandwidth is usually higher with smaller data types and hw often have special blocks for efficient multiply-accumulate operations with smaller data types only (e.g. XMX on GPU and AMX on CPU don't support fp32) + less memory I/O operations due to smaller tensor byte size.